inst/markdown/compareScenariosLimes/cs2_main.Rmd

```diff
@@ -204,6 +204,19 @@ largeRegions <- function(df) {
   return(df %>% filter(region %in% largeRegionsVec))
 }
 
+removeAggRegions <- function(df, except = NULL){
```

Comment:
Would it be useful to add an argument that takes the regions that should be removed, e.g.
```suggestion
removeAggRegions <- function(df, remove = c("EU27", "EUETS", "EUETS_nonDE", "World") , except = NULL){
```